pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -51,9 +51,12 @@ buildDotnetModule rec {
     ln --force --symbolic "${lib.getExe _7zzWithOptionalUnfreeRarSupport}" src/ArchiveManagement/NexusMods.FileExtractor/runtimes/linux-x64/native/7zz
```

Comment:
Is this still needed? If so does it still need `--force`?

Reply:
And can it use `lib.getExe`?

Reply:
I think having the exe on the PATH should be enough, so I guess just list the package in `propagatedBuildInputs`?

Reply:
Good catch! I don't think it's even needed at build time, so I've added it to `runtimeDeps`.